dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -131,6 +132,10 @@ std::tuple<std::optional<RegionRetryList>, RegionException::RegionReadStatus> Ma
                 if (batch_cop)
                     status = RegionException::RegionReadStatus::NOT_FOUND;
             });
+            fiu_do_on(FailPoints::random_trigger_remote_read, {
+                if (batch_cop)
+                    status = RegionException::RegionReadStatus::NOT_FOUND;
+            });
```

Comment:
It seems these three failpoint can be merged.

Reply:
These three failpoints have their own uses.
The first failpoint will only be triggered once.
The second failpoint will always be triggered,
The third failpoint is triggered randomly according to probability

Reply:
Hmmm. Is it necessary to use these three failpoints in one test at the same time? If not, keeping one would be enough.